### ISSUE TITLE
MisconfigurationException when monitoring val_loss during training

### ISSUE THREAD

**Issue by ssterrett** [paninski-lab/lightning-action#26]
### Description
Training runs for one epoch and then fails with a MisconfigurationException because the ModelCheckpoint callback cannot find the val_loss metric, even though a validation loop appears to have executed.
### Error Message
MisconfigurationException: `ModelCheckpoint(monitor='val_loss')` could not find the monitored key in the returned metrics: ['lr-Adam', 'val_accuracy', 'val_f1', 'train_accuracy', 'train_f1', 'train_loss', 'epoch', 'step']. HINT: Did you call `log('val_loss', value)` in the `LightningModule`?
### Steps to Reproduce
Run the training command: model.train(output_dir='/path/to/your/outputdir')
The error occurs at the end of the first epoch during the ModelCheckpoint callback execution.
### Expected Behavior
Training to log the validation loss under the key 'val_loss'. 
### Environment
Python Version: 3.10
OS: Linux via Docker
lightning-action: v0.2.4

**Comment by themattinthehatt** [comment]
@ssterrett I just wanted to check in and see if this was still an issue for you after training the model with more data (as we discussed via email).

**Comment by ssterrett** [comment]
hi, yes training with more data as you suggested fixed the bug! you can close this.
